charts/nextcloud/values.yaml

```
@@ -146,7 +146,15 @@ nextcloud:
   #          )
   #        )
   #      );
-
+  # Hooks for auto configuration
+  # Here you could write small scripts which are placed in `/docker-entrypoint-hooks.d/<hook-name>/<hook-name>.sh`
```

```suggestion
  # Here you could write small scripts which are placed in `/docker-entrypoint-hooks.d/<hook-name>/helm.sh`
```
They are always mounted like this, no?

---

Correct (i change it after writing docu)